environment/simulation/simulation.py

```diff
@@ -6,9 +6,9 @@
 import pymunk.autogeometry
 import pymunk.pygame_util
 
-import utils.simulation_utils as utils
-import utils.simulation_pymunk_utils as pymunk_utils
```

Comment:
Nie wiem co tu się odjaniepawliło ale github mi pokazuje diffa do innego pliku niż ten na masterze. Nie rozumim. Ale ogólnie to te importy to już ja zrobiłem. Domyślam się po prostu że ten PR powstał przed moimi zmianami ale usuń to żeby nie nadpisało.